my-project/src/Controller/ProductCategoryController.php

```diff
@@ -203,6 +204,30 @@ public function UploadAndAddImageToCategory(Request $request, ProductCategory $p
         return new JsonResponse();
     }
 
+    /**
+     * @Route("/{image}/{category}", name="remove_image_from_category", methods="DELETE")
+     */
+    public function removeImageFromCategory(Request $request, Image $image, ProductCategory $category): Response
+    {
+
+            try
+            {
+                $em = $this->getDoctrine()->getManager();
+                $category->getImages()->removeElement($image);
+                $em->flush();
+                $this->addFlash(
+                    'notice',
+                    'Operation complete'
+                );
+            }
+            catch(\Exception $exception)
+            {
```

Comment:
@mSobonVirtua  maybe it would be better to trigger less general error for such operations?